modules/ensemble/lib/framework/widget/toast.dart

```diff
@@ -28,6 +28,10 @@ class ToastController {
     return _instance;
   }
 
+  void closeToast() {
+    _toast.removeQueuedCustomToasts();
```

Comment:
this will only get rid of the queued ones, what about the present one? should there be also be call like this - 

`fToast.removeCustomToast()`

also there is a method called `Fluttertoast.cancel()` that gets rid of all of them, did you try that?

Reply:
No, I thought the same at first, but then I looked at the code and we are using FToast and not Fluttertoast, so the package provides two different classes. And due to that I was using the FToast one, and the _toast is the object of the same. Also, I tested and it does remove all the toasts

Reply:
got it, I see that the OverlayEntry.remove will get rid of the overlay. Thanks for explaining.